ext/RMagick/rmimage.c

```diff
@@ -10327,7 +10327,7 @@ Image_opaque_channel(int argc, VALUE *argv, VALUE self)
 #if defined(IMAGEMAGICK_7)
     exception = AcquireExceptionInfo();
     BEGIN_CHANNEL_MASK(new_image, channels);
-    okay = OpaquePaintImage(new_image, &target_pp, &fill_pp, MagickFalse, exception);
+    okay = OpaquePaintImage(new_image, &target_pp, &fill_pp, invert, exception);
```

Comment:
@Watson1978 is there any way for us to test for this?

Reply:
I think we have to check the appearance of the image...